src/ignorefile/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ocaml/opam2:alpine as builder
```

Comment:
Currently everything should put into top level Dockerfile, that Dockerfile will call `make build` on your source folder.
You may need some magic there for multi-stage build.

Reply:
Multi-stage build with make is quite tricky. It is much easier if we can let each checker to build their own image. Say the docker image should have an self-contained executable at `/checker`. In the outermost dockerfile, we just have to use multi-stage build to copy all executables to build a final image. The problem is about how to package python program as a self-contained executable. This tool seems promising https://github.com/facebookincubator/xar Maybe we should refactor the current build architecture before adding new checkers.

Reply:
Just copy the binary might not work unless you do all static link.